data/5e/wizards_of_the_coast/supplement/unearthed_arcana/races_of_eberron/ua_abilities.lst

```diff
@@ -162,9 +162,9 @@ Light Step				KEY:RoE Warforged ~ Skirmisher ~ Light Step			SORTKEY:1_Race ~ War
+Darkwood core (unarmored)	KEY:RoE Warforged ~ Integrated Protection ~ Darkwood core		PREMULT:2,[!PREABILITY:1,CATEGORY=Special Ability,TYPE.AC_Tracker_Choice],[PREABILITY:1,CATEGORY=Special Ability,RoE Warforged ~ Integrated Protection]		SORTKEY:1_Race ~ Warforged ~ 1i1	PRERACE:1,Warforged																	CATEGORY:Special Ability	TYPE:Warforged ~ Integrated Protection.SpecialQuality.Defense.AC_Tracker.AC_Tracker_Choice.AC_Tracker_SelectionMade	DESC:11 + your Dexterity modifier (add proficiency bonus if proficient with light armor)	BONUS:VAR|AC_Total|AC_Base				BONUS:VAR|AC_Base|11|TYPE=Base	BONUS:VAR|AC_Total|AC_Shield|PREVAREQ:AC_ShieldDisabled,0	BONUS:VAR|AC_Total|Proficiency_Bonus|PREABILITY:1,CATEGORY=Internal,Armor Proficiency ~ Light	COST:0
+Composite plating (armor)	KEY:RoE Warforged ~ Integrated Protection ~ Composite plating	PREMULT:2,[!PREABILITY:1,CATEGORY=Special Ability,TYPE.AC_Tracker_Choice],[PREABILITY:1,CATEGORY=Special Ability,RoE Warforged ~ Integrated Protection]	SORTKEY:1_Race ~ Warforged ~ 1i1	PRERACE:1,Warforged	PRETEXT:Medium armor proficiency	PREABILITY:1,CATEGORY=Internal,Armor Proficiency ~ Medium	CATEGORY:Special Ability	TYPE:Warforged ~ Integrated Protection.SpecialQuality.Defense.AC_Tracker.AC_Tracker_Choice.AC_Tracker_SelectionMade	DESC:13 + your Dexterity modifier (maximum of 2) + your proficiency bonus			BONUS:VAR|AC_Total|AC_Base+Proficiency_Bonus	BONUS:VAR|AC_Base|13|TYPE=Base	BONUS:VAR|AC_Total|AC_Shield|PREVAREQ:AC_ShieldDisabled,0	BONUS:VAR|AbilityScoreLimit|2|TYPE=Base										COST:0	
+Heavy plating (armor)		KEY:RoE Warforged ~ Integrated Protection ~ Heavy plating		PREMULT:2,[!PREABILITY:1,CATEGORY=Special Ability,TYPE.AC_Tracker_Choice],[PREABILITY:1,CATEGORY=Special Ability,RoE Warforged ~ Integrated Protection]		SORTKEY:1_Race ~ Warforged ~ 1i1	PRERACE:1,Warforged	PRETEXT:Heavy armor proficiency	PREABILITY:1,CATEGORY=Internal,Armor Proficiency ~ Heavy	CATEGORY:Special Ability	TYPE:Warforged ~ Integrated Protection.SpecialQuality.Defense.AC_Tracker.AC_Tracker_Choice.AC_Tracker_SelectionMade	DESC:16 + your proficiency bonus; disadvantage on Dexterity (Stealth) checks			BONUS:VAR|AC_Total|AC_Base+Proficiency_Bonus	BONUS:VAR|AC_Base|16|TYPE=Base	BONUS:VAR|AC_Total|AC_Shield|PREVAREQ:AC_ShieldDisabled,0	BONUS:VAR|DisableDexForAC|1|TYPE=Boolean										COST:0	
```

Comment:
Why are we breaking the PRExxx tags here to keep the ability normal looking in the UI?

Reply:
Without that change, the options don't show up. It looks like they depend on themselves. I couldn't choose any of the options when at least the darkwood core option should have been available. With this change and no armor specialization, only darkwood was available. When I added a class that had medium armor, composite showed up.

Reply:
I'll accept this and then deep dive on what is going wrong later. Likely the base ability has a TYPE that's blocking.

Reply:
Maybe I'm confused, but aren't these the specific "instances" of the "integrated protection" ability, and the PREABILITY that I modified is just saying that we have the "base" ability, and then along with the later PRETEXT allows for enabling which versions are allowed based on which armor proficiency you have?